packages/backend/package.json

```diff
@@ -116,7 +116,7 @@
         "migrate-production": "knex migrate:latest --knexfile dist/database/knexfile.js",
         "seed": "knex seed:run --knexfile src/database/knexfile.ts",
         "seed-production": "knex seed:run --knexfile dist/database/knexfile.js",
-        "rollback-last": "knex migrate:rollback --knexfile src/database/knexfile.ts",
```

Comment:
`knex migrate:rollback` is to rollback the last batch of migrations. eg: If you migrated everything in 1 go, it would rollback everything. 

We just want to rollback the last migration script.